server/src/ScriptRunner/Scripts/index/transfer_reason_table.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS public.trasfer_reason
+(
+
+    id integer NOT NULL,
+    display_name character varying COLLATE pg_catalog."default",
+    CONSTRAINT trasfer_reason_pk PRIMARY KEY (id)
+);
+INSERT INTO public.trasfer_reason(id, display_name)
+	VALUES (0,'אחר')
```

Comment:
you can insert all the values in one command
https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-insert-multiple-rows/